### PR TITLE
Ground work for notifications in UI

### PR DESCRIPTION
**Pull request by jchorl**

### CODE REVIEW

File: client/src/actions/notification.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+function guid() {
```
Comment (matthewdu) [inline client/src/actions/notification.js]: Can we not use an atomic int?

Comment (jchorl) [inline client/src/actions/notification.js]: We don't even need atomicity. We can use a monotonically incrementing int. I was debating that vs. guid, dont think it matters really. I'll change it.

File: client/src/components/App/index.js [hunk 2]
```diff
@@ -6,8 +6,12 @@ import ImmutablePropTypes from 'react-immutable-proptypes';
 import Home from '../Home';
 import Login from '../Login';
 import Nav from '../Nav';
+import Notifications from '../Notifications';
 import { fetchUser } from '../../actions/user';
 import './App.css';
+// TODO remove
```
Comment (rwongone) [inline client/src/components/App/index.js]: Is this TODO part of your WIP or for later?

Comment (jchorl) [inline client/src/components/App/index.js]: removed

**Pull request merged**